unit-tests/src/test/java/org/eclipse/collections/impl/PersonAndPetKataTest.java

```diff
@@ -410,14 +408,12 @@ public void groupPeopleByLastName()
     @Test
     public void groupPeopleByLastNameUsingStreams()
     {
-        Map<String, List<Person>> byLastNameStream =
-                this.people.stream().collect(
-                        Collectors.groupingBy(Person::getLastName));
+        Map<String, List<Person>> byLastNameStream = this.people.stream()
```

Comment:
should .stream() be on a new line?

Reply:
Doesn't matter much to me. collect() only takes a single parameter, so I thought it was weird to wrap that like it was a list. What do you think?